util/Migrator/DbMigrator.cs

```diff
@@ -105,10 +109,10 @@ private void PrepareDatabase(CancellationToken cancellationToken = default)
     }
 
     private bool MigrateDatabase(bool enableLogging = true,
-        bool repeatable = false,
-        string folderName = MigratorConstants.DefaultMigrationsFolderName,
-        bool dryRun = false,
-        CancellationToken cancellationToken = default)
+    bool repeatable = false,
```

Comment:
ℹ️ Your formatting seems to be going crazy. Just run `dotnet format` locally and it will fix all this.

Reply:
ran
![image](https://github.com/user-attachments/assets/2c561a27-1da2-4714-b8eb-d0aa62094081)